NetSSL_Win/cmake/PocoNetSSLWinConfig.cmake

```diff
@@ -2,4 +2,5 @@ include(CMakeFindDependencyMacro)
 find_dependency(PocoFoundation)
 find_dependency(PocoUtil)
 find_dependency(PocoNet)
+find_dependency(OpenSSL REQUIRED COMPONENTS SSL)
```

Comment:
NetSSL_Win does not depend on OpenSSL

Reply:
Do I have to fix it somehow, or is it ok?

Reply:
it's ok, already manually merged